services/client-app/src/routes/demo.api.tsx

```
@@ -24,7 +24,9 @@ function RouteComponent() {
   if (isError) {
     return <div>Error loading assets: {error.message}</div>
   }
-
+  {
+    console.log('data.items', data?.items)
+  }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)